rules/cyberark/privilege_escalation_cyberarkpas_error_audit_event_promotion.toml

```diff
@@ -0,0 +1,61 @@
+[metadata]
+creation_date = "2021/06/23"
+maturity = "production"
+updated_date = "2021/06/23"
+min_stack_version = '7.14.0'
+
+[rule]
+author = ["Elastic"]
+description = "Identifies the occurrence of a CyberArk Privileged Access Security (PAS) error level audit event."
+false_positives = ["To tune this rule, add exceptions to exclude any event.code which should not trigger this rule."]
+from = "now-30m"
+index = ["filebeat-*", "logs-cyberarkpas.audit*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "CyberArk Privileged Access Security Error"
+note = """## Config
+
+The cyberarkpas Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule.
+
+## Triage and analysis
+
+This is a promotion rule for CyberArk error events, which are alertable events per the vendor.
+Consult vendor documentation on interpreting specific events.
+"""
+references = [
+    "https://docs.cyberark.com/Product-Doc/OnlineHelp/PAS/Latest/en/Content/PASREF/Vault%20Audit%20Action%20Codes.htm?tocpath=Administration%7CReferences%7C_____3",
```

Comment:
do we need the `?tocpath=Administration%7CReferences%7C_____3` param?

Reply:
I know it is awful, but yes it is a specific reference